sources/packages/web/src/views/aest/NoticeOfAssessment.vue

```diff
@@ -0,0 +1,49 @@
+<template>
+  <v-container>
+    <div class="mb-4">
+      <header-navigator
+        title="Assessment"
+        subTitle="Notice of Assessment"
+        :routeLocation="{
+          name: AESTRoutesConst.ASSESSMENTS_SUMMARY,
+          params: { applicationId, studentId },
+        }"
+      />
+    </div>
+    <full-page-container>
+      <notice-of-assessment-form-view :assessmentId="assessmentId" />
+    </full-page-container>
+  </v-container>
```

Comment:
same here on v-container.

Reply:
Sorry, I did not follow this one. Is it not the same from the previous one?

Reply:
Adjusted!